src/java/org/apache/cassandra/index/sai/disk/io/RAMIndexOutput.java

```diff
@@ -70,11 +75,16 @@ public void writeTo(IndexOutput externalOut) throws IOException
         externalOut.writeBytes(out.getBytes(), 0, out.getPosition());
     }
 
-    public BytesRef getBytes()
+    public BytesRef getBytesRef()
     {
         return new BytesRef(out.getBytes(), 0, out.getPosition());
     }
 
+    public void writeTo(byte[] bytes, int offset)
```

Comment:
Nit: the offset argument passed by the callers is always zero, so perhaps this could be simplified to `writeTo(byte[] bytes)`. Also, I would place the method right after the other `writeTo(IndexOutput)` method, so similar methods stay together.